Lib/test/test_pep646_syntax.py

```diff
@@ -317,10 +319,10 @@
 
 __test__ = {'doctests' : doctests}
 
-def test_main(verbose=False):
-    from test import support
-    from test import test_pep646_syntax
-    support.run_doctest(test_pep646_syntax, verbose)
+def load_tests(loader, tests, pattern):
```

Comment:
Would moving the `doctest` import into this function improve performance?

Reply:
I don't see how.